TWT/apps/challenges/views/home.py

```diff
@@ -108,10 +107,13 @@ def get(self, request: WSGIRequest) -> HttpResponse:
         timers = Timer.objects.all()
         if len(list(timers)) != 0:
             timer = timers[0]
-            context['timer_title'] = timer.title
-            context['timer_date'] = str(timer.date_time.date())
-            context['timer_time'] = ':'.join(str(timer.date_time.time()).split(':')[0:-1])
-        
+            if timer.date_time < timezone.now():
```

Comment:
`timezone.utcnow()`?

Reply:
@mohamed040406 i mean its set as utc in settings but for being extra sure i can change.... i checked and it returns utc time only but ill change it

Reply:
@mohamed040406 
![image](https://user-images.githubusercontent.com/66492212/121907036-4e021c80-cd49-11eb-9872-0898c088df23.png)
![image](https://user-images.githubusercontent.com/66492212/121907059-53f7fd80-cd49-11eb-86cb-95fbe1291c27.png)
there is no timezone.utcnow()
but according to docs and our settings it will be utc and I checked in the console as well
![image](https://user-images.githubusercontent.com/66492212/121907135-683bfa80-cd49-11eb-8930-2ed69f59cf53.png)